src/renderers/webgl-fallback/WebGLBackend.js

```diff
@@ -1125,33 +1127,45 @@ class WebGLBackend extends Backend {
 
 	createBindings( bindGroup, bindings ) {
 
-		this.updateBindings( bindGroup, bindings );
+		if ( this._knownBindings.has( bindings ) === false ) {
 
-	}
+			this._knownBindings.add( bindings );
 
-	updateBindings( bindGroup, bindings ) {
+			let uniformBuffers = 0;
+			let textures = 0;
 
-		if ( ! bindGroup ) return;
+			for ( const bindGroup of bindings ) {
 
-		const { gl } = this;
+				this.set( bindGroup, {
+					textures: textures,
+					uniformBuffers: uniformBuffers
+				} );
 
-		const bindingsData = this.get( bindings );
-		const bindGroupData = this.get( bindGroup );
+				for ( const binding of bindGroup.bindings ) {
 
-		if ( bindingsData.textureIndex === undefined ) bindingsData.textureIndex = 0;
+					if ( binding.isUniformBuffer ) uniformBuffers ++;
+					if ( binding.isSampledTexture ) textures ++;
 
-		if ( bindGroupData.textureIndex === undefined ) {
+				}
 
-			bindGroupData.textureIndex = bindingsData.textureIndex;
+			}
 
-		} else {
+		}
 
-			// reset textureIndex to match previous mappimgs when rebuilt
-			bindingsData.textureIndex = bindGroupData.textureIndex;
+		this.updateBindings( bindGroup, bindings );
 
-		}
+	}
+
+	updateBindings( bindGroup /*, bindings*/ ) {
+
+		if ( ! bindGroup ) return;
```

Comment:
Nit: When the `return` is executed, is `bindGroup` then `undefined` or `null`? I would favor a more explicit check like:
```js
if ( bindGroup === undefined ) return;
```

Reply:
I'll check, I remember some examples triggered this path.